### ISSUE TITLE
Clarify language around community mediation/CoC issues

### ISSUE THREAD

**Issue by austinlparker** [open-telemetry/community#1842]
Currently there is ambiguous language in the charter (https://github.com/open-telemetry/community/blob/main/governance-charter.md and https://github.com/open-telemetry/community/blob/main/community-membership.md#responsibilities-and-privileges-3 for example) around who ultimately mediates and resolves CoC or other community issues in the project. Per consensus, this responsibility rests with the GC. 

AI: Review existing language and update as needed

**Comment by bogdandrutu** [comment]
Also https://github.com/open-telemetry/community/pull/1840

**Comment by danielgblanco** [comment]
Agreed. In particular,

>Escalate reviewer, approver, maintainer and other workflow concerns (i.e. responsiveness, availability, and general contributor community health) to the TC.

I believe this should be split into two bullet points, one containing the aspects to be escalated to the TC (i.e. technical direction or mediation) and another one for the aspects to be escalated to the GC (i.e. SIG health, CoC). I'll mark this as accepted, because although that may not be the agreed change, I believe it's something we should clarify.

cc @jpkrohling

**Comment by mtwo** [comment]
@danielgblanco are you still intending to generate a PR for this?

**Comment by danielgblanco** [comment]
Yes. Thanks for the ping. I'll raise a PR before the end of next week (I'm travelling this week).
